build.gradle

```diff
@@ -219,7 +219,8 @@ subprojects { subproject ->
         testCompileOnly 'org.apiguardian:apiguardian-api:1.0.0'
 
         testCompile group: 'com.github.tomakehurst', name: 'wiremock', version: '2.18.0'
-        testCompile group: 'com.opentable.components', name: 'otj-pg-embedded', version: '0.12.0'
+        testCompile "org.testcontainers:postgresql:1.12.3"
```

Comment:
Moved to testcontainers from embedded postgres (otj-pg-embedded).
Looks like otj-pg-embedded is not compatible with latest version of liquibase-core as it fails to create sequence for column type "serial" . Similar to the problem mentioned here https://liquibase.jira.com/browse/CORE-1816. I've tried bumping `otj-pg-embedded` to latest but no luck. The error occurs against otj-pg-embedded only.

Eg: Caused by: org.postgresql.util.PSQLException: ERROR: null value in column "id" violates not-null constraint Detail: Failing row contains (null, TEST2, user@hmcts.net, TEST, workBasketDefaultCaseType, workBasketDefaultState, 2019-11-18 23:17:04.184981, UPDATE, rtaashrie9j1otx@example.com).